installer/scripts/omsadmin.sh

```diff
@@ -796,6 +820,36 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    local result_limit="No Limit"
+    if [ ! -f $PROC_LIMIT_CONF ]; then
+        log_error "Limits configuration file '$PROC_LIMIT_CONF' missing!"
+        return 1
+    fi
+    local count=`get_current_omsagent_proc_limit | wc -l`
+
+    if [ "$count" -eq "0" ]; then
+        log_info "There is NO present limit to number of processes for $AGENT_USER."
+    else
+        if [ "$count" -gt "1" ]; then
+            log_info "$count $AGENT_USER entries found in $PROC_LIMIT_CONF."
+            log_info ">>Last entry should apply<<" # NOTE:  This will not pick up -1 entries.
+            get_current_omsagent_proc_limit
+            log_info "End of the $count duplicate entry info Message"
+        fi
+        result_limit=`get_current_omsagent_proc_limit | tail -1 | awk '{print $4}'`
+        if [ "$result_limit" -eq "-1" ]; then
+            result_limit="No Limit"
+            log_info "There is NO present limit to number of processes for $AGENT_USER."
```

Comment:
I think the messaging for this case should be slightly different than the messaging on line 833 when there are no lines in limits.conf for omsagent, since this case represents that a limit is set, but the "limit" is infinity. Perhaps "The number of allowed processes for $AGENT_USER is unlimited"?

Reply:
Ok.